packages/workflow/src/utils.ts

```diff
@@ -1,3 +1,5 @@
+import * as ErrorReporterProxy from './ErrorReporterProxy';
```

Comment:
Based on usage I see, we often import while aliasing `LoggerProxy as Logger` to shorten the calls, which are so frequent.